api/docs.go

```diff
@@ -727,6 +727,13 @@ const docTemplate = `{
                     "items": {
                         "type": "string"
                     }
+                },
+                "uuids": {
+                    "description": "List of RepositoryConfig uuids to serach",
```

Comment:
```suggestion
                    "description": "List of RepositoryConfig UUIDs to search",
```

Not sure why RepositoryConfig is one word.